src/style.css

```diff
@@ -1,3 +1,91 @@
+* {
+  box-sizing: border-box;
+}
```

Comment:
The universal selector (*) is used here, which can impact performance. It's better to set box-sizing to border-box for specific elements that require it, or use a more targeted selector.